packages/utils/src/fhir/bundle/find_with_custom_filter.sh

```
@@ -0,0 +1,164 @@
+    echo "  Ubuntu/Debian: sudo apt-get install jq"
+    echo "  CentOS/RHEL: sudo yum install jq"
+    exit 1
+fi
+
+# Function to check if a file matches the custom filter
+check_file_with_filter() {
+    local file="$1"
+    local filter="$2"
+    
+    # Apply the custom jq filter to the file
+    local result=$(jq -e "$filter" "$file" 2>/dev/null)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)